### ISSUE TITLE
image not fulscreen on ipad

### ISSUE THREAD

**Issue by softwarebam** [PeterStaev/nativescript-image-swipe#57]
can set image width ro height ?

**Comment by PeterStaev** [comment]
Hey @softwarebam , the image is automatically resized so that it fills in the lowest dimension of the screen. May be you need the same as #25 ?

**Comment by softwarebam** [comment]
It seem not resize image to fill in the lowest dimension of screen in some devices.
I find the issue on iPhone 11 (iOS 13) and iPad air2 (iOS 13) as shown below. 
![05FD2D60-0D06-42CA-86C2-232A649ECA25](https://user-images.githubusercontent.com/39789188/74426316-b4daa180-4e87-11ea-917e-d6d8d549dfe5.jpg)

The devices need to change orientation, so the image is resized and fill in the lowest dimension of screen properly.


I use nativescript 6.4.

**Comment by PeterStaev** [comment]
Is the image high enough resolution? If it is not then it will not work, since the plugin sets the max possible scale to be 1.0:
https://github.com/PeterStaev/nativescript-image-swipe/blob/5d0284009c6c34fe7ad113299446cf740ac13039/image-swipe.ios.ts#L274-L277

**Comment by softwarebam** [comment]
Yes, the image is too small. Is it possible to auto zoom to fill in the lowest dimension of screen?

**Comment by PeterStaev** [comment]
Currently this is not possible, marking it as an enhancement as it makes sense to have an option to allow such behavior.
